README.md

```diff
@@ -1,15 +1,15 @@
 **Theme Name:** Twenty Twenty  
 **Theme URI:** https://wordpress.org/themes/twentytwenty/  
 **Contributors:** the WordPress team  
-**Requires at least:** WordPress 4.9.6  
+**Requires at least:** WordPress 4.7  
```

Comment:
Leaving a note here to remind myself to reconfirm this required version later in the project to ensure it is correct